### ISSUE TITLE
Add contributors map

### ISSUE THREAD

**Issue by aboullaite** [fbdevelopercircles/open-source-edu-bot#37]
#### Is your feature request related to a problem? Please describe.
We've been discussing in #34 #26 some ways to include contributors. relying on HoF and All contributors bot won't be possible in the near future, since those bots can't be added to organizations. 

#### Describe the solution you'd like
What I suggest is adding a YAML file where each contributor/open source learner will add his details, and then use this yaml file to display list of contributors, or even a map from where each contributor is located

With this in place, we can help first time contributors to easily make their first PR, and make the bot guide them in the process.

#### Documentation
The yaml file should look something like:
```yaml
contributors:
   - name: Mohammed Aboullaite
     github: @aboullaite 
     city: Casablanca
     country: Morocco
```
Then we can make a web page that parse this file and renders contributors list.

#### What's next
- [ ] Add Yaml file to the repo
- [ ] Add steps in the bot that guide users on how to contribute and add their PRs
- [ ] Add web page listing all contributors
- [ ] Explore options to add it to readme as well (Github Actions ?)

**Comment by mohitacecode** [comment]
is making web page is still pending if yes I would love to make it. :) @aboullaite

**Comment by aboullaite** [comment]
Yes @mohitacecode it is! Please feel free to work on it, your contribution is very welcomed. Thanks

**Comment by ankitaggarwal23** [comment]
i would like to contribute

**Comment by ankitaggarwal23** [comment]
contributors.yaml updated #79

**Comment by vj-codes** [comment]
@aboullaite any resources for UI guidelines for Facebook led web pages?

**Comment by vj-codes** [comment]
@aboullaite @caabernathy @elinguiuriel 
I found there's a tool to automate contributors https://github.com/all-contributors/all-contributors-cli .
It's pretty easy to [install](https://allcontributors.org/docs/en/bot/installation) and a good solution for future .But it can be implemented by repo maintainers only :(
Let me know if something like this is allowed !

**Comment by aboullaite** [comment]
@vj-codes we already discussed it in #26! Its unfortunately not possible at the moment

**Comment by vj-codes** [comment]
Yeah right completely missed seeing that sorry my bad

**Comment by mohitacecode** [comment]
> is making a web page is still pending if yes I would love to make it. :) @aboullaite
> 
> PS : I am new here and would love to contribute to this project in any way.

Hey @aboullaite, I was gone for too long sorry about that I have some quick questions:

Is there any code for web pages for any other thing where we will add the contributor's web page?

**Comment by aboullaite** [comment]
@mohitacecode I didn't get it? what do you mean ?
